clients/drcachesim/tools/external/example/empty.cpp

```diff
@@ -111,5 +110,6 @@ empty_t::process_memref(const memref_t &memref)
 bool
 empty_t::print_results()
 {
+    std::cout << "Empty tool results:" << std::endl;
```

Comment:
Tools print to stderr by convention (for online use where output is mixed with app output, to distinguish from the app).